services/QuillLMS/client/app/bundles/PremiumHub/containers/UsageSnapshotsContainer.tsx

```diff
@@ -239,7 +240,12 @@ const UsageSnapshotsContainer = ({ adminInfo, accessType, }) => {
   function handleClickDownloadReport() { window.print() }
 
   function mapItemsIfNotAll(selectedItems, allItems, mapKey = 'id') {
-    if (unorderedArraysAreEqual(selectedItems, allItems)) return null
+    // selectedItems may, by design, be a superset of allItems, but if everything in allItems is in selectedItems, we want to treat it as "everything" being selected
+    const allItemsSelected = allItems.every((i) => {
```

Comment:
nit, but I always read `i` as `index`.

Reply:
Good call-out.  Worth making that clearer.